config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-presubmits-main.yaml

```diff
@@ -386,6 +386,9 @@ presubmits:
           exit $result
         securityContext:
           privileged: true
+    annotations:
+      testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
```

Comment:
the unit test failed with 

```
could not write config: 1 error occurred:
	* found duplicate name after normalizing: (DashboardTab) capzprcoveragemain

```

🤔 

is there another job using capz-pr-coverage-main as tab-name?

Reply:
Here - [https://github.com/kubernetes/test-infra/blob/cc33f69d38c1ed4f47a1355c9cec34e827a4535f/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml#L47-L49](https://github.com/kubernetes/test-infra/blob/cc33f69d38c1ed4f47a1355c9cec34e827a4535f/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml#L47-L49)

Reply:
Also I see test_group for the same - [https://github.com/kubernetes/test-infra/blob/cc33f69d38c1ed4f47a1355c9cec34e827a4535f/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml#L64-L70](https://github.com/kubernetes/test-infra/blob/cc33f69d38c1ed4f47a1355c9cec34e827a4535f/config/testgrids/kubernetes/sig-cluster-lifecycle/config.yaml#L64-L70)